04/app.js

```diff
@@ -2,7 +2,36 @@ console.log('DOM');
 
 // struktura do wykorzystania w pętli
 const menuItems = [
-    {text: 'start', url: '/'},
-    {text: 'galeria', url: '/gallery'},
-    {text: 'kontakt', url: '/contact'},
-];
\ No newline at end of file
+	{ text: 'start', url: '/' },
+	{ text: 'galeria', url: '/gallery' },
+	{ text: 'kontakt', url: '/contact' },
+];
+
+const nav = document.querySelector('nav');
+
+const ul = document.createElement('ul');
+
+const li1 = document.createElement('li');
+const li2 = document.createElement('li');
+const li3 = document.createElement('li');
+
+const a1 = document.createElement('a');
+const a2 = document.createElement('a');
+const a3 = document.createElement('a');
```

Comment:
Tworzenie tych elementów powinno być robione wew. `forEach`.
Inaczej nie jest to elastyczne rozwiązanie. Dodanie nowej pozycji w menu psuje całe rozwiązanie...